internal/storage/conn.go

```diff
@@ -59,7 +62,19 @@ func InitConn(params ConnParams) *Connection {
 func (c *Connection) Connect() error {
 	dsn := MakeDSN(c.params)
 	c.logger.LogF(monitor.F{"dsn": dsn}).Info("connecting to the DB")
-	db, err := sqlx.Connect(c.dialect, dsn)
+	var err error
+	var db *sqlx.DB
+	for i := 0; i < maxDBConnectAttempts; i++ {
+		db, err = sqlx.Connect(c.dialect, dsn)
+		if err == nil {
+			break
+		}
+		secondsToWait := i + 1
+		c.logger.Log().Warning("Database Connection Err: ", err)
```

Comment:
A bit of a nitpick, but those two log entries are not guaranteed to reach the physical log stream together. Why not put it all in a single message?

Reply:
No particular reason, besides I thought you would want them separated out because one is an error and one is an info statement. You never specified what you wanted, so I had to assume brother. so I went with a warning message because the error was causing a failure. I don't really have a strong opinion. So be specific and I will change it to whatever you want it to be.

Reply:
A single `Warning` is fine here.